history/characters/48000_tirassian.txt

```diff
@@ -13801,4 +13801,67 @@
 		religion = holy_light
 	}
 	620.4.21={ death=yes }
+}
+
+#Wallace Boltscrew, non-canon
+49500={
+	name=Wallace
+	dynasty=26116
+	dna="oojckdfngkd"
+	culture=tirassian religion=tidemother
+	martial=4 diplomacy=5 stewardship=6 intrigue=5 learning=7
+	add_trait=grey_eminence
+	564.4.25={ birth=yes add_trait=creature_human }
+	583.1.1={
+		employer = 62080 # Colton
+	}
+	603.1.1={
+		culture = pirate_culture
+		add_spouse=63100 # Anetta
```

Comment:
No wife.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/81499526-55046300-92dd-11ea-8276-938ed38f530b.png)

</details>